spec/support/fake_cronitor.rb

```diff
@@ -12,18 +16,24 @@ class FakeCronitor < Sinatra::Base
   post '/v1/monitors' do
     payload = JSON.parse request.body.read
     # Check that we have the necessary payload values very simply
-    %w(name rules notifications).each do |k|
+    %w[name rules notifications].each do |k|
```

Comment:
I ran rubocop over the file to find problems, and accidentally auto-fixed them. Will revert.